python/ray/air/_internal/device_manager/utils.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
+from ray.air._internal.device_manager.hpu import HPUTorchDeviceManager
+from ray.air._internal.device_manager.npu import (
+    NPU_TORCH_PACKAGE_AVAILABLE,
+    NPUTorchDeviceManager,
+)
+from ray.air._internal.device_manager.nvidia_gpu import CUDATorchDeviceManager
+from ray.air._internal.device_manager.torch_device_manager import TorchDeviceManager
+
+logger = logging.getLogger(__name__)
+
+
+ACCELERATOR_TORCH_DEVICE_MANAGER = {
+    "GPU": CUDATorchDeviceManager,
+    "HPU": HPUTorchDeviceManager,
+    "NPU": NPUTorchDeviceManager,
+}
+
+
+def try_register_torch_accelerator_module() -> None:
+    try:
+        if NPU_TORCH_PACKAGE_AVAILABLE:
+            import torch_npu  # noqa: F401
+
+        if HPU_PACKAGE_AVAILABLE:
+            import habana_frameworks.torch.hpu as torch_hpu  # noqa: F401
+
+    except ImportError:
+        raise ImportError("Could not import PyTorch")
+
+
+def get_torch_device_manager_cls_by_resources(resources) -> TorchDeviceManager:
```

Comment:
Instead of determining the manager cls by a dict of resource, can we just directly look up `ACCELERATOR_TORCH_DEVICE_MANAGER` by the accelerator type?

Reply:
In my view, the resources specified by the user for the worker may not be singular, but could also be a combination of resources like `{'CPU': 1, 'GPU': 1}`. 
So it is necessary to iterate through the elements of the dictionary, and this step would be more appropriate to do in the utility function. What are your thoughts on this?